server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -208,6 +209,21 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 		const deltasCollectionName = config.get("mongo:collectionNames:deltas");
 		const scribeCollectionName = config.get("mongo:collectionNames:scribeDeltas");
 
+		// Setup for checkpoint collection
+        const deliLocalCheckpointEnabled = config.get("deli:localCheckpointEnabled");
+        const scribeLocalCheckpointEnabled = config.get("scribe:localCheckpointEnabled");
+
+		const operationsDb = await operationsDbMongoManager.getDatabase();
+		const checkpointsCollection =
+			operationsDb.collection<core.ICheckpoint>(checkpointsCollectionName);
+		await checkpointsCollection.createIndex(
+			{
+				documentId: 1,
+				tenantId: 1,
+			},
```

Comment:
I dodn't think you need this compound index, compound index is long used for sorting by two columns.  but you are not sorting the two